selfdrive/car/toyota/interface.py

```diff
@@ -355,16 +355,19 @@ def update(self, c, can_strings):
     # events
     events = self.create_common_events(ret)
 
-    if self.CS.low_speed_lockout and self.CP.openpilotLongitudinalControl:
-      events.add(EventName.lowSpeedLockout)
-    if ret.vEgo < self.CP.minEnableSpeed and self.CP.openpilotLongitudinalControl:
-      events.add(EventName.belowEngageSpeed)
-      if c.actuators.accel > 0.3:
-        # some margin on the actuator to not false trigger cancellation while stopping
-        events.add(EventName.speedTooLow)
-      if ret.vEgo < 0.001:
-        # while in standstill, send a user alert
-        events.add(EventName.manualRestart)
+    if self.CS.CP.openpilotLongitudinalControl:
+      if self.CS.brake_hold:
+        events.add(EventName.brakeHold)
```

Comment:
Move this to common events?

Reply:
yes, added to carState for this https://github.com/commaai/cereal/commit/aa349f6c314b1fcc7f5a42e6f4bfe0d466335634